src/cache/cache_las.c

```diff
@@ -75,11 +75,15 @@ __wt_las_config(WT_SESSION_IMPL *session, const char **cfg)
 		    "max cache overflow size %" PRId64 " below minimum %d",
 		    cval.val, WT_LAS_FILE_MIN);
 
+	/* This is expected for in-memory configurations. */
+	las_session = S2C(session)->cache->las_session[0];
```

Comment:
Could you add an assertion in here:

`WT_ASSERT(session, las_session != NULL || F_SET(conn, WT_CONN_IN_MEMORY));`

Reply:
4d28e8b
Oops, forgot this one.
Also changed `F_SET` to `F_ISSET` since I think that's making the flag true.